src/render/dom/utils/filter-props.ts

```diff
@@ -3,6 +3,16 @@ import { isValidMotionProp } from "../../../motion/utils/valid-prop"
 
 let shouldForward = (key: string) => !isValidMotionProp(key)
 
+export type IsValidProp = (key: string) => boolean
+
+export function loadExternalIsValidProp(emotionIsPropValid?: IsValidProp) {
+    if (!emotionIsPropValid) return
+
+    // Handle events explicitly as Emotion validates them all as true
+    shouldForward = (key: string) =>
+        key.startsWith("on") ? !isValidMotionProp(key) : emotionIsPropValid(key)
```

Comment:
If there's an injected `isValidProp` provided by `MotionConfig`, wouldn't it be hijacked by `isValidMotionProp`? For example, `isValidProp={key => key !== "onStart"}`

Reply:
I think that might have to be a limitation of the API that our events are always consumed